docs/user-guide/cli-using-team-configuration-team-leaders.md

```diff
@@ -1,23 +1,24 @@
-# Team configuration for team leaders
+# Configuration for team leaders
```

Comment:
@adam-wolfe @zFernand0 @sbohnert - please do not review this page b/c it's not going into the V3 site's final version. This content was broken up and re-organized into other pages included in this PR.